src/renderer.ts

```diff
@@ -289,6 +315,29 @@ export class MagicMoveRenderer {
         el.classList.add(CLASS_ENTER_FROM)
         el.classList.add(CLASS_ENTER_ACTIVE)
 
+        const elRect = el.getBoundingClientRect()
+        const pos = { x: elRect.x - anchorRect.x, y: elRect.y - anchorRect.y }
+
+        magicMoveElements.push({
```

Comment:
I am a bit worried about the extra complexity here, where we are constructing data only for a hook we don't use. This means we are not guarding the implementation and can be complicated to maintain.

Could you come up with a more organic solution for your case?